Rainfall-prediction/raindata.py

```diff
@@ -0,0 +1,79 @@
+# importing libraries
+import pandas as pd
+import numpy as np
+import sklearn as sk
```

Comment:
You can remove this line as it's not used anywhere.

Reply:
on it!

Reply:
@Ashlin21dev Only `import sklearn as sk`

Reply:
i ran into a problem sir!!

Reply:
@Ashlin21dev What's the problem?